layers/sflow_test.go

```diff
@@ -13,6 +13,59 @@ import (
 	"github.com/google/gopacket"
 )
 
+// Test packet collected from PCAP from SolarWinds tool. See the test below for contents
+var SFlowTestPacketWithIPv4Data = []byte{
```

Comment:
Can you trim the test data to include IPv4 sample only, no counter sample? It should be easy to do so. You just need to modify a few bytes.

Reply:
Ack